src/app/docs/character-count/character-count-docs.component.html

```diff
@@ -7,6 +7,52 @@
     The character count indicator component extends a text input to apply a character limit and display an indicator with the number of characters entered and the character limit.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <div
+      style="width: 50%;"
```

Comment:
What is our "standard" here? I see differences accross our components:
- Datepicker is set to `400px` 
- Country Field goes 100%
- Here we are doing 50%

Should we have a standard for this?

Reply:
That's fine for this. We can adjust if needed later.